src/cache/cache.service.ts

```diff
@@ -44,34 +50,89 @@ export class CacheService<T extends unknown> {
     return join(this.cacheDir, networkDir);
   }
 
-  private async getCacheFilePath(): Promise<string> {
+  private getCacheFileName(batchIndex: number): string {
+    return `${batchIndex}.${this.cacheFile}`;
+  }
+
+  private async getCacheFilePaths(): Promise<string[]> {
     const dir = await this.getCacheDirPath();
-    return join(dir, this.cacheFile);
+    const existCacheFiles: string[] = [];
+
+    let batchIndex = 0;
+
+    do {
+      try {
+        const filePath = join(dir, this.getCacheFileName(batchIndex));
```

Comment:
Perhaps we should use glob to find the files we need and exclude others?

Reply:
replaced with `glob`